package-lock.json

```diff
@@ -4154,9 +4154,9 @@
       "dev": true
     },
     "flow-bin": {
-      "version": "0.74.0",
-      "resolved": "https://registry.npmjs.org/flow-bin/-/flow-bin-0.74.0.tgz",
-      "integrity": "sha512-tIN9J5qg71S4UbofCu80tve8a+p7Hj7ytwUtu79cLg9KJVVTNnVVJXKgCghVzaZT1Rvl9SMHVPlDs9uYhPHEGQ==",
+      "version": "0.75.0",
+      "resolved": "https://registry.npmjs.org/flow-bin/-/flow-bin-0.75.0.tgz",
+      "integrity": "sha1-uW0e6Z07RGoyJr5mtAEyJM6d8mA=",
```

Comment:
I believe the latest NPM uses `sha512` hashes exclusively. Or does it normally use a mix? I've found I had to delete `node_modules` and clean the NPM cache, then `npm install`, to fix this.

Reply:
@edemaine There are issues of `npm` mixing integrity hashes, https://github.com/npm/npm/issues/16938 and https://github.com/npm/npm/issues/17749, closed but people are still reporting problems.

Reply:
Shouldn't we try to get sha512 hashes in there?  Again, I've found resetting NPM as listed above (thanks to these various issues) has fixed the problem.